doc/calculator/Cargo.toml

```
@@ -1,12 +1,15 @@
 [package]
 name = "calculator"
-version = "0.20.1"
+version = "0.20.2"
 authors = ["Niko Matsakis <niko@alum.mit.edu>"]
-workspace = "../.." # <-- We added this and everything after!
+workspace = "../.."                             # <-- We added this and everything after!
```

```suggestion
workspace = "../.." # <-- We added this and everything after!
```

Feel free to ignore those if you have a good reason to add all those spaces.